### ISSUE TITLE
spring-cloud-starter-oauth2 issue with spring-cloud-services-dependencies

### ISSUE THREAD

**Issue by louisthomas** [pivotal-cf/spring-cloud-services-starters#92]
My project is using spring-cloud-services-starter-config-client and spring-cloud-starter-oauth2. I try to configure oauth2 client (client credentials grant type) by adding  @EnableOAuth2Client annotation. My IDE (IntelliJ) is not able to find it. ![screenshot](https://github.com/louisthomas/spring-boot-2-zuul/raw/master/images/enableOauth2Client_issue.png)

When I remove this dependency management, it's working.
````xml
        <dependency>
                <groupId>io.pivotal.spring.cloud</groupId>
                <artifactId>spring-cloud-services-dependencies</artifactId>
                <version>${spring-cloud-services.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
````

Versions:
Spring-boot.version: 2.1.4.RELEASE
spring-cloud-services.version: 2.1.2.RELEASE
spring-cloud.version: Greenwich.SR1

Have you seen this issue before?

Here my [github repo](https://github.com/louisthomas/spring-boot-2-zuul)

**Comment by louisthomas** [comment]
Compare Maven dependency tree:
![compare](https://github.com/louisthomas/spring-boot-2-zuul/raw/master/images/compareMavenDependencyTree.png)


* [Maven dependency tree without spring-cloud-services-dependencies](https://github.com/louisthomas/spring-boot-2-zuul/blob/master/without.txt)
* [Maven dependency tree with spring-cloud-services-dependencies](https://github.com/louisthomas/spring-boot-2-zuul/blob/master/with.txt)

**Comment by habuma** [comment]
As it turns out, this is already resolved by #86, although it wasn't obvious to me at first that was the case. Also, the fix for #86 has not yet been released, but that is imminent (and will be in SCS starters 2.1.3.RELEASE when it is available).

In the meantime, you can work around this problem by explicitly specifying the Spring Security OAuth2 dependency in your build with the following:

```
<dependency>
	<groupId>org.springframework.security.oauth</groupId>
	<artifactId>spring-security-oauth2</artifactId>
	<scope>compile</scope>
</dependency>
```

**Comment by habuma** [comment]
2.1.3.RELEASE is now available and will soon be the version chosen through the Initializr. You may try it by changing the value of the `spring-cloud-services.version` property in the build to 2.1.3.RELEASE. Closing this as resolved.
